.NET/Microsoft.Recognizers.Text.DateTime/Arabic/Parsers/ArabicSetParserConfiguration.cs

```diff
@@ -168,5 +168,12 @@ public bool GetMatchedUnitTimex(string text, out string timex)
 
         public string WeekDayGroupMatchString(Match match) => SetHandler.WeekDayGroupMatchString(match);
 
+        public string ReplaceValueInTextWithFutTerm(string text, string value)
+        {
```

Comment:
As this behaviour is the same everywhere, the method should be declared in a shared place and receive the term or terms as a parameter. Also, there may be more than one term in the list, so the method should act accordingly.

Reply:
I added functionality to replace a value with only one term from a list; I didn't want to define a new pattern, so I used an existing variable.

Reply:
Replace function with one common method.